### ISSUE TITLE
PDB with multiple CHAIN

### ISSUE THREAD

**Issue by dallouche** [MattMcPartlon/AttnPacker#3]
Hi ,   would you mind to tell me how to take into account , input pdb with more than 1 chain ?

i tried to used AttnPacker , with 1DQZ for instance:
the input pdb includes : 

curated_pdbs/1DQZ.pdb<-pdb :2 chains total
== chain A
1-280
#AA 279
== chain B
503-782
#AA 279

AttnPacker output profile scp only for the chain A.
1DQZ_attn.pdb<-pdb :1 chains total
== chain A
1-277
#AA 276

only 1 chain with 277 AA is inferred in the output 
 The quality assessment analysis seems to have the same behavior ( only the  first chain seems to be analyzed)    

would you mind to tell me how to   deal with pdb with several chain ? 
 
regards david

**Comment by dallouche** [comment]
The inference has been done with the sneapet provided in the jupiter example (cf) :
prediction = runner.infer(
        pdb_path=pdb_path, 
            #Boolean Tensor indicating which residues to design
        seq_mask=None, 
            #Whether to format output (process into logits, seq labels, etc. or return raw no-selde and pair output)
        format=True,
            #Chunk inference by successively packing smaller crops of size chunk_size
            #This allows packing of arbitrarily long proteins
        chunk_size = 500,
    )
    print(f"Ran Inference on {runner.device} in time {round(time.time()-start,2)} seconds")
    predicted_protein = make_predicted_protein(model_out = prediction['model_out'], seq = prediction['seq'])
        # Save predicted PDB to this location
    #pdb_out_path = f"./{predicted_protein.name}_packed.pdb"
    pdb_out_path = model_pdb
    print(f"saving pdb to {pdb_out_path}")
        # write predicted protein with plddt score in b-factor column
    predicted_protein.to_pdb(pdb_out_path, beta=prediction["pred_plddt"].squeeze())

**Comment by MattMcPartlon** [comment]
Hi David,

I am AFK right now. The pdb parsing code only extracts only the first chain (by default). I will add an option to parse all chains and send you an update tomorrow when it's ready.

Best,
Matt

**Comment by dallouche** [comment]
Hi Matt, i didn't receive the  code with the new option  
regard David

**Comment by aboob2** [comment]
Hi Matt, is there any update on using AttnPacker with PDB files that have multiple chains?
